integer/src/lib.rs

```diff
@@ -694,6 +696,22 @@ pub fn binomial<T: Integer + Clone>(mut n: T, k: T) -> T {
     r
 }
 
+/// Calculate the multinomial coefficient.
+///
+/// Panics if no integers are specified.
```

Comment:
Must we panic?  The formula is a simple product, so it seems natural to use the identity `1` for the empty case.